go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/couchbaselabs/try-cb-golang
+
+go 1.15
+
+require (
+	github.com/couchbase/gocb/v2 v2.1.8
+	github.com/couchbase/gocbcore/v8 v8.0.5
```

Comment:
This is slightly concerning, we've been on v9 for a while now. Why is anything in the travel-sample-app using gocbcore directly?

Reply:
Ah, I was trying to check for the error type here, that's probably why:

https://github.com/couchbaselabs/try-cb-golang/pull/2/commits/ced747beb3b35ec9ba00c993fa3dd40a0f949cd5#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R344

Reply:
Ah ha, you don't need to use the errors from gocbcore. You should just use the corresponding ones from gocb.